src/components/styles/join.scss

```diff
@@ -50,33 +51,28 @@
     flex-direction: column;
     align-items: center;
     justify-content: flex-end;
-
     overflow: hidden;
 
-    &.management {
-      background-image: url("../../images/join/management.jpg");
+    &.board3 {
+      background-image: url("../../images/join/management.png");
```

Comment:
maybe use https://www.gatsbyjs.com/plugins/gatsby-background-image/ or use a `<StaticImage/>` element instead of background-image. These would allow for gatsby to optimize these images better for the web.